packages/docusaurus/src/server/plugins/init.ts

```diff
@@ -114,7 +115,7 @@ For more information, visit https://v2.docusaurus.io/docs/using-plugins.`);
         version: pluginVersion,
       };
     })
-    .filter(Boolean);
+    .filter(<T>(item: T): item is Exclude<T, null> => Boolean(item));
```

Comment:
🤪

Reply:
will merge this, but if you can add a utility fn in docusaurus-utils that would be appreciated :)

https://kentcdodds.com/blog/typescript-function-syntaxes#type-guards